src/app/http-stalker.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {TestBed} from '@angular/core/testing';
+
+import {HttpStalkerService} from './http-stalker.service';
+
+describe('HttpStalkerService', () => {
+  let service: HttpStalkerService;
```

Comment:
mi è venuto in mente ora: una convenzione molto carina di cui avevo letto è di dichiarare esplicitamente la cosa che si sta testando chiamandola `sut` (System Under Test). 

questo rende molto semplice capire cosa va mockato e cosa no

Reply:
In teoria ogni volta quello che viene testato è component o service a seconda di cosa viene testato mentre gli altri hanno nomi più articolati, però anche questa è carina